opentok/opentok.py

```diff
@@ -1262,11 +1282,17 @@ def set_broadcast_layout(self, broadcast_id, layout_type, stylesheet=None):
 
         :param String stylesheet optional: CSS used to style the custom layout.
         Specify this only if you set the type property to 'custom'
+
+        :param String screenshare_type optional: Layout to use for screenshares. Must
+        set 'layout_type' to 'bestFit'
         """
         payload = {
             "type": layout_type,
         }
 
+        if screenshare_type is not None:
+            payload["screenshareType"] = screenshare_type
```

Comment:
Not sure if it's worth validating in the SDK level, but one gotcha (which ultimately led to me not using this) is that you can't actually use this with any layout_type but stylesheet.

I guess by not enforcing it here, no SDK update is needed _if_ that changes...

Reply:
It actually can't be used with any `type` other than `bestFit`. It looks like the API is defaulting to that if you only pass a `screenshareType`. `stylesheet` is only used for the custom layout, which is mutually exclusive with `screenshareType` (and the API will return an error that can be caught)